src/js/components/FileInput/FileInput.js

```diff
@@ -67,6 +67,7 @@ const FileInput = forwardRef(
       multiple,
       name,
       onChange,
+      onClickRemove,
```

Comment:
Suggestion lets rename it and call as `canRemove` or something like this, where can accept a boolean or a function that returns a boolean

Reply:
Recommending `onRemove` instead. I think just accepting a function is sufficient.